internal/state/graph/gateway_listener.go

```diff
@@ -197,7 +197,7 @@ func createAddressesValidator(gw *v1beta1.Gateway) listenerValidator {
 		if len(gw.Spec.Addresses) > 0 {
 			path := field.NewPath("spec", "addresses")
 			valErr := field.Forbidden(path, "addresses are not supported")
-			return []conditions.Condition{conditions.NewListenerUnsupportedAddress(valErr.Error())}
+			return []conditions.Condition{conditions.NewListenerUnsupportedValue(valErr.Error())}
```

Comment:
do we need to reflect this in the commit/PR name?

Reply:
Sure, I can add it to the commit once I squash

Reply:
it seemed unrelated to 0.7.0

Reply:
It's related. See "The UnsupportedAddress Listener condition reason..." item under "Status Changes" https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.7.0